ocs_ci/ocs/platform_nodes.py

```diff
@@ -266,37 +266,54 @@ def restart_nodes(self, nodes, force=True, timeout=300, wait=True):
         Args:
             nodes (list): The OCS objects of the nodes
             force (bool): True for Hard reboot, False for Soft reboot
-            timeout (int): time in seconds to wait for node to reach 'not ready' state,
-                and 'ready' state.
+            timeout (int): time in seconds to wait for node to reach READY state
             wait (bool): True if need to wait till the restarted OCP node
                 reaches READY state. False otherwise
 
         """
+        num_events_pre_reboot, num_events_post_reboot = ([] for i in range(2))
         vms = self.get_vms(nodes)
+        node = nodes[0]
```

Comment:
Is this variable useful ?

Reply:
This was used with reboot_events_cmd. Since it has been moved inside the loop, removed this line